ios/Classes/Convert.swift

```diff
@@ -325,6 +325,16 @@ class Convert {
         if let draggable = options["draggable"] as? Bool {
             delegate.isDraggable = draggable
         }
+    
+        if let geometry = options["geometry"] as? [[Double]], geometry.count > 0 {
```

Comment:
On line creation this will deserialize the geometry twice - once here and once when creating MGLLineStyleAnnotation.
So maybe add this as a seperate function that can be used in `"line#add"` to create the geometry for MGLLineStyleAnnotation as well as in  the  `"line#update"`